packages/react-spinner/src/components/Spinner/useSpinnerStyles.ts

```diff
@@ -231,6 +234,9 @@ const useTrackStyles = makeStyles({
   primary: {
     ['& > svg > circle.fui-Spinner__Tail']: {
       stroke: tokens.colorBrandStroke1,
+      '@media screen and (-ms-high-contrast: active)': {
```

Comment:
The standard version of the query should be used rather than the `-ms` prefixed one.
```suggestion
      '@media (forced-colors: active)': {
```